apps/testfront/package.json

```diff
@@ -6,11 +6,11 @@
   "type": "module",
   "scripts": {
     "dev": "vite -c vite.config.dev.ts",
-    "dev:ssr": "ts-node --experimentalSpecifierResolution node --esm server/server",
-    "build": "npm run build:client && npm run build:ssr && npm run build:netlify",
+    "dev:ssr": "ts-node server/server",
+    "prod:ssr": "npm run build && npm run dev:ssr --omit=dev",
+    "build": "npm run build:client && npm run build:ssr",
     "build:client": "vite build",
-    "build:ssr": "vite build -c vite.config.server.ts",
-    "build:netlify": "vite build -c vite.config.netlify.ts"
```

Comment:
Did we drop this file? (It seems like it is not needed anymore)

Reply:
Yep, removed it for now.